### PR TITLE
Client app flight select

### PR DESCRIPTION
**Pull request by ArinaSavchenko**
Screenshot 2020-10-02 153728
Screenshot 2020-10-02 153756

### CODE REVIEW

File: Airline_Web_API/Airlina_DB/Airlina_DB.sqlproj [hunk 1]
```diff
@@ -67,4 +67,7 @@
     <Build Include="dbo\Tables\Tickets.sql" />
     <Build Include="dbo\Tables\TicketTypes.sql" />
   </ItemGroup>
+  <ItemGroup>
+    <RefactorLog Include="Airlina_DB.refactorlog" />
+  </ItemGroup>
```
Comment (andrei-kruhliakou-itechart) [inline Airline_Web_API/Airlina_DB/Airlina_DB.sqlproj]: You've deleted file from the folder on disk, but not form the project, You'll get errors again if you try to sync the database.

File: Airline_Web_API/Airline_Web_API/Models/TicketType.cs [hunk 2]
```diff
@@ -9,13 +9,13 @@ public class TicketType
...
         public string Name { get; set; }
-        public int? CarryOnBagNumber { get; set; }
+        public int? CarryOnBagsNumber { get; set; }
         public int? CarryOnBagMaxWeight { get; set; }
         public int? BaggageNumber { get; set; }
         public int? BaggageMaxWeight { get; set; }
         public bool SeatReservation { get; set; }
         public bool Changes { get; set; }
-        public string Refund { get; set; }
+        public bool? Refund { get; set; }
```
Comment (andrei-kruhliakou-itechart) [inline Airline_Web_API/Airline_Web_API/Models/TicketType.cs:18]: Why would you need null value here? What is the meaning of it?

File: Airline_Web_API/Airline_Web_API/ViewModels/TicketsViewModel.cs [hunk 3]
```diff
@@ -9,6 +9,7 @@ public class TicketsViewModel
     {
         public int Id { get; set; }
         public decimal Price { get; set; }
+        public int flightId { get; set; }
```
Comment (andrei-kruhliakou-itechart) [inline Airline_Web_API/Airline_Web_API/ViewModels/TicketsViewModel.cs]: Property name should start with capital letter

File: ClientApp/src/app/Models/Airport.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 export interface Airport{
-  Id?: number;
+  id?: number;
   name?: string;
   city?: string;
   country?: string;
```
Comment (andrei-kruhliakou-itechart) [inline ClientApp/src/app/Models/Airport.ts]: Why are all those fields nullable?

File: ClientApp/src/app/Services/tickets.service.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  httpOptions = {
+    headers: new HttpHeaders({'Content-Type': 'application/json'})
+  };
+
+  constructor(private http: HttpClient) {
+  }
+
+  getTicket(id: number): Observable<Ticket> {
+    // tslint:disable-next-line:no-debugger
+    debugger;
```
Comment (andrei-kruhliakou-itechart) [inline ClientApp/src/app/Services/tickets.service.ts]: Please do not leave comments and debug information in code

File: ClientApp/src/app/flight-search-results/flight-search-results.component.css [hunk 6]
```diff
@@ -53,7 +54,7 @@ div.tickets {
   background-color: #f3fab7;
 }
 
-p.header, .flightInfo, .ticketTypeName {
+p.header, .flightInfo, .ticketTypeName, .time, .price, .ticketType {
```
Comment (andrei-kruhliakou-itechart) [inline ClientApp/src/app/flight-search-results/flight-search-results.component.css]: Please put alternative selectors (i.e. separated with commas) from the new line

**Review by andrei-kruhliakou-itechart — CHANGES_REQUESTED** [review]

File: Airline_Web_API/Airline_Web_API/Models/TicketType.cs (hunk 2 above)
Comment (ArinaSavchenko) [reply Airline_Web_API/Airline_Web_API/Models/TicketType.cs:18]: Null - if there is no refund for this type of ticket, false - refund for fee, true - free refund

Comment (andrei-kruhliakou-itechart) [reply Airline_Web_API/Airline_Web_API/Models/TicketType.cs:18]: I'm approving it for now, but please consider using more appropriate data type here instead.

**Review by andrei-kruhliakou-itechart — APPROVED** [review]

**Pull request merged**